src/containers/stage-header.jsx

```diff
@@ -45,7 +46,7 @@ StageHeader.propTypes = {
     isFullScreen: PropTypes.bool,
     isPlayerOnly: PropTypes.bool,
     onSetStageUnFull: PropTypes.func.isRequired,
-    stageSize: PropTypes.oneOf(Object.keys(STAGE_SIZES)),
+    stageSize: PropTypes.oneOf(Object.values(STAGE_SIZES)),
```

Comment:
Using `Object.keys` here was intentional, since we want to refer to the values by its key until the value is needed.  I don't think it's consequential in the way that we're using it, but it can have an effect under some optimization circumstances, where the keys of objects are renamed by the optimizer, so you want to use keys rather than values (which are untouched by the optimizer) as much as possible.

Reply:
I switched it to `Object.values` because in an earlier version I had `largeConstrained: 'large-constrained'` in the `STAGE_SIZES` enum and that was causing errors. Isn't `Object.values` correct, though, even considering that the keys might be renamed by the optimizer? For example:
```js
const myEnum = {key1: 'value1', key2: 'value2'}
var myProp = myEnum.key1
console.log(myProp); // logs "value1"
```

Reply:
Well either keys or values will work, just will take up different amounts of space depending on  the optimization.

You're right though. We want the prop to be the value, so, you know, we can read it...   `<Component prop={myEnum.KEY} />` not `<Component prop={"KEY"} />` 🤕✋ (face palm)

Reply:
Good news! They added this emoji with Unicode 9.0: 🤦